Description: Disable unpackaged yet empfindung and dependent test
Last-Update: 2022-11-15
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -39,8 +39,8 @@
 version = "0.4"
 default-features = false
 
-[dev-dependencies.empfindung]
-version = "0.2"
+#[dev-dependencies.empfindung]
+#version = "0.2"
 
 [dev-dependencies.lab]
 version = ">=0.4, <1.0"
--- a/src/test.rs
+++ b/src/test.rs
@@ -39,7 +39,7 @@
     }
 }
 
-/// Tries all colours in the 256-colour ANSI palette and chooses one with
+/*// Tries all colours in the 256-colour ANSI palette and chooses one with
 /// smallest ΔE*₀₀ to `rgb(y, y, y)`.
 fn best_grey(y: u8) -> u8 {
     let grey = [y, y, y];
@@ -66,7 +66,7 @@
     for i in 0..256 {
         assert_eq!(best_grey(i as u8), to_ansi((i as u8, i as u8, i as u8)));
     }
-}
+}*/
 
 /// Tests that getting value for grey colour given as RGB triple and one given
 /// as just shade of grey produce the same result.
